config/environments/development.rb

```diff
@@ -76,6 +76,15 @@
   require 'byebug'
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
-  config.action_mailer.delivery_method = :letter_opener
-  config.action_mailer.perform_deliveries = true
+  #config.action_mailer.delivery_method = :letter_opener
+  #config.action_mailer.perform_deliveries = true
```

Comment:
it's easier to develop with :letter_opener, so i suggest to leave here :letter_opener, and remove smtp from development.rb